Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/configuration/update/CCLConfigurationUpdater.kt

```diff
@@ -57,9 +56,29 @@ class CCLConfigurationUpdater @Inject constructor(
     @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
     internal suspend fun updateConfiguration(): Boolean {
         return coroutineScope {
-            val newBoosterRulesDownloaded = async { boosterRulesRepository.update() }
-            val newCclConfigDownloaded = async { cclConfigurationRepository.updateCCLConfiguration() }
-            newBoosterRulesDownloaded.await() or newCclConfigDownloaded.await()
+            val boosterRulesDeferred = async { boosterRulesRepository.update() }
+            val cclConfigDeferred = async { cclConfigurationRepository.updateCCLConfiguration() }
+
+            val boosterRulesResult = boosterRulesDeferred.await()
+            val cclConfigResult = cclConfigDeferred.await()
+
+            updateExecutionTimeOnSuccess(boosterRulesResult, cclConfigResult)
+
+            val newBoosterRules = (boosterRulesDeferred.await() == BoosterRulesRepository.UpdateResult.UPDATE)
+            val newCclConfig = (cclConfigDeferred.await() == CCLConfigurationRepository.UpdateResult.UPDATE)
```

Comment:
You already have
```kotlin 
    val boosterRulesResult = boosterRulesDeferred.await()
    val cclConfigResult = cclConfigDeferred.await()
```
can we use them here ?

Reply:
Oh... fixed.